org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -4406,11 +4680,35 @@ class WorkspaceEdit {
 	}
 }
 
+
+/**
+ * The options of a Workspace Symbol Request.
+ */
+@JsonRpcData
+class WorkspaceSymbolOptions implements WorkDoneProgressOptions {
+	Boolean workDoneProgress
```

Comment:
Similar remark as above: Should we have a base type with the field declaration and the other `*Options` should inherit the field from that one?

Reply:
This is basically the same remark as above for the `*Options`